packages/clay-empty-state/src/index.tsx

```diff
@@ -136,11 +136,7 @@ const ClayEmptyState = ({
 			)}
 
 			<div className="c-empty-state-title">
-				<span className="text-truncate-inline">
-					<span className="text-truncate">
-						{title || defaultTile}
-					</span>
-				</span>
+				<span>{title || defaultTile}</span>
```

Comment:
@ilzamcmed mostly looks good, just need to run `yarn format` to fix this SF issue.

Reply:
Thanks @pat270! 
All fixed